content/docs/docs-nav.yaml

```diff
@@ -16,6 +16,7 @@ nav:
         - Designating admins: '/docs/getting-started/create-admin-group/'
         - Modelling Users and Groups: '/docs/getting-started/teams/'
       - Adding plugins: '/docs/getting-started/configuring-backstage-plugins/'
+      - Create a Proxy: '/docs/getting-started/create-proxy/'
```

Comment:
This should probably be a part of the adding a plugin sidebar item and the scorecard item. You wouldn't create a proxy for the sake of it - it would always be in the context of one of them

Reply:
The idea is that you would link to this page from all the places you would want to have a proxy in place. Then we create curated examples for each link on this page.